### PR TITLE
feat(testing): adds jest and unit tests for client

### PR DESCRIPTION
**Pull request by componentDidMount**
- adds jest testing framework
- adds unit tests for Client, TopicClient, WorkflowClient, JobClient
- adds jest setup which runs and manages the latest zeebe docker instance
- camunda/zeebe:latest must have been pulled at least once before the tests can run: `docker pull camunda/zeebe:latest`

### CODE REVIEW

File: lib/Client.js [hunk 1]
```diff
@@ -35,9 +35,18 @@ class Client {
   _initClient (options) {
     const clientOptions = Object.assign(clientDefaultOptions, options)
 
-    const status = this.ffiLib.zbc.InitClient(this.ffiLib.structs.newGoString(clientOptions.bootstrapAddr))
+    const status = JSON.parse(
```
Comment (componentDidMount) [inline lib/Client.js]: Had to add the parse so line 44 can work, otherwise status is just a string.

**Comment by componentDidMount** [comment]
@felix-mueller @menski to try out, you should have the latest zeebe docker image on your machine  (or pull it with `docker pull camunda/zeebe:latest`) and then just run "npm test" in the repository main directory.

**Comment by componentDidMount** [comment]
If you are funky you can also run with visible coverage: `npm test -- --coverage`

```sh
-------------------------|----------|----------|----------|----------|-------------------|
File                     |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
-------------------------|----------|----------|----------|----------|-------------------|
All files                |    75.96 |    62.03 |    68.42 |    76.97 |                   |
 zbc-node                |      100 |      100 |      100 |      100 |                   |
  index.js               |      100 |      100 |      100 |      100 |                   |
 zbc-node/lib            |    77.86 |    67.69 |       80 |    77.86 |                   |
  Client.js              |    95.45 |    66.67 |     87.5 |    95.45 |               123 |
  FFILib.js              |    73.33 |       50 |       60 |    73.33 |       42,50,61,64 |
  JobClient.js           |    61.36 |    38.89 |    71.43 |    61.36 |... 12,113,116,117 |
  TopicClient.js         |       92 |    91.67 |      100 |       92 |             57,58 |
  WorkflowClient.js      |    79.41 |    80.77 |      100 |    79.41 |... 66,71,73,88,89 |
 zbc-node/lib/__internal |    69.05 |    35.71 |    46.15 |    72.97 |                   |
  Structs.js             |    73.33 |      100 |       50 |    73.33 |       36,42,45,48 |
  errors.js              |      100 |      100 |      100 |      100 |                   |
  options.js             |      100 |      100 |      100 |      100 |                   |
  utils.js               |    52.63 |    35.71 |       40 |    57.14 | 53,60,67,81,88,95 |
-------------------------|----------|----------|----------|----------|-------------------|
```

**Comment by felix-mueller** [comment]
Nice contribution!
Just a question before I merge this: Did you use http://standardjs.com/ for code styling? Just asking because I see a lot of whitespace changes etc.

**Comment by componentDidMount** [comment]
@felix-mueller I amended my previous commit with the format changes and script I showed you, feedback is welcome.

**Pull request merged**